Makefile

```diff
@@ -21,36 +21,44 @@
    SMP_FLAGS = -j$(shell nproc)
  endif
 
- ISYS_ROOT = $(shell ./bootstrap/coriolisEnv.py --query-isys-root)
- INST_ROOT = $(shell ./bootstrap/coriolisEnv.py --query-inst-root)
- WORK_ROOT = ${HOME}/coriolis-2.x/Linux.el9/Release.Shared
+ ISYS_ROOT    = $(shell ./bootstrap/coriolisEnv.py --query-inst-root)
+ SRC_ROOT     = $(shell ./bootstrap/coriolisEnv.py --query-src-root)
```

Comment:
Are these actually needed any more?

Reply:
In case of an install in the user's account. Typically for developers, yes. So they don't need to choose a build and installation location of their own (and it's automated).